### ISSUE TITLE
Settings are not saved

### ISSUE THREAD

**Issue by technicbasics** [trevorsandy/lpub3d#692]
### Subject
No settings are saved.
Render settings (FOV,...) and other settings are also affected.
It looks like the program always starts with the default settings. I also tried to run the program as administrator, which usually solves these problems with mobile versions. Without success. In general, the program seems very unstable. It crashes after about every 5 steps.

### Environment
* Version of LPub3D - 2.4.6r98 64bit
* Operating system - Win11 64-bit

### Configuration
[Note: Provide details and content below as needed with appropriate redactions. To produce the Windows registry extract for an installed distribution, go to the command console and enter the command line shown. The extract file will be placed on your desktop.]
* LPub3D installation path
  - Windows: `C:\LPub3D (mobile version)

First start
![2023-03-26 09_57_48-Pistenbully 600 Polar mpd - LPub3D v2 4 6 r98 (Release) png_first start](https://user-images.githubusercontent.com/49073781/227763757-0f47da74-e68b-46ea-a84e-cc2a7d935aea.png)

Second start
![2023-03-26 09_57_48-Pistenbully 600 Polar mpd - LPub3D v2 4 6 r98 (Release) png_second start](https://user-images.githubusercontent.com/49073781/227763759-0d516812-7ace-4120-a564-146bf4628173.png)

**Comment by trevorsandy** [comment]
Thank you for reporting this behaviour.

Did this behaviour appear with v2.4.6 r98 ?

Are you able to consistently reproduce an AbEnd ?

Can you provide the LPub3D log ?

Cheers,

**Comment by technicbasics** [comment]
Hi Trevor
I have now tried the r57.
There this problem with the settings is not present.
The crash of the program is with me with the Native Renderer more often than with the LDView Renderer. At the beginning the program crashed immediately after I used the command "0 !LPUB SUBMODEL_DISPLAY SHOW LOCAL TRUE". (Native) By default the command at the beginning of the file is set to "0 !LPUB SUBMODEL_DISPLAY SHOW GLOBAL FALSE", and when I want to show a (sub)model, I set it Local to true. Furthermore I noticed with the r98 that not always all pages are loaded/counted and then only some pages are displayed in the dropdown at the top of the header. After a "right click" and "Refrssh page image" all pages are displayed again.
I hope I attached the right .log files.
I have also attached you a file with my header settings.
Regards

[LPub3DLog.txt](https://github.com/trevorsandy/lpub3d/files/11071868/LPub3DLog.txt)
[LPub3DLog.txt.zip](https://github.com/trevorsandy/
[main.txt](https://github.com/trevorsandy/lpub3d/files/11071871/main.txt)
lpub3d/files/11071869/LPub3DLog.txt.zip)
